src/app.rs

```diff
@@ -67,6 +67,11 @@ pub fn build() -> App<'static, 'static> {
                 .multiple(true)
                 .help("Display extended file metadata as a table"),
         )
+        .arg(
+            Arg::with_name("no-config")
+                .long("no-config")
+                .help("Do not read a configuration file"),
```

Comment:
Feels like this line seems to convey that we have options to pass config file?
How about something like "Ignore configuration file" or even "Do not read configuration file"?

Reply:
I will have a look at changing that to something along the lines of "ignore-config" and update the description accordingly. That makes more sense.